### DISCUSSION TITLE
Will the `OptionalPaths.swift` be moved to the `CasePaths` package?

### DISCUSSION THREAD

**Discussion by grdsdev** [pointfreeco/isowords#143] [Q&A] [answered]
Hey guys, awesome work you all doing here.

Just asking, there are any plans to move https://github.com/pointfreeco/isowords/blob/main/Sources/TcaHelpers/OptionalPaths.swift to the `CasePaths` package?

I need them in a project, for now I just copied the whole file, but would be nice if gets integrated into the CasePaths package.

Thanks and keep doing the great work.

**Comment by stephencelis** [comment, marked as answer]
Hey @grsouza! We definitely want to get something like OptionalPath into CasePaths, but it's a big API to introduce and we want to make sure we get it right. We're experimenting with some alternatives, as well, and will hopefully have enough experience with the problem to address it early next year. In the meantime, we recommend to copy the file into projects as needed :smile:

I'm going to convert this to a discussion, thanks for taking the time to post!
